packages/arb-token-bridge-ui/src/token-bridge-sdk/BridgeTransferStarter.ts

```diff
@@ -52,6 +52,10 @@ export type TransferProps = {
   amount: BigNumber
   signer: Signer
   destinationAddress?: string
+  overrides?: {
```

Comment:
@dewanshparashar I'm still thinking whether we can do this in a better way? Right now we have `overrides` on `TransferProps` which would contain all possible overrides for different transfer types.